contracts/Gateway/NativeTokenGateway.sol

```diff
@@ -6,28 +6,38 @@ import { SafeERC20, IERC20 } from "@openzeppelin/contracts/token/ERC20/utils/Saf
 import { ReentrancyGuard } from "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 
 import { IWrappedNative } from "./Interfaces/IWrappedNative.sol";
-import { INativeTokenGateway, IVToken } from "./INativeTokenGateway.sol";
+import { INativeTokenGateway } from "./INativeTokenGateway.sol";
+import { IVToken } from "./Interfaces/IVtoken.sol";
 
 /**
  * @title NativeTokenGateway
  * @author Venus
- * @notice NativeTokenGateway contract facilitates interactions with a vToken market for native tokens (Native or wrappedNativeToken)
+ * @notice NativeTokenGateway contract facilitates interactions with a vToken market for native tokens (Native or wNativeToken)
  */
 contract NativeTokenGateway is INativeTokenGateway, Ownable2Step, ReentrancyGuard {
     using SafeERC20 for IERC20;
 
     /**
      * @notice Address of wrapped ether token contract
      */
-    IWrappedNative public immutable wrappedNativeToken;
+    IWrappedNative public immutable wNativeToken;
+
+    /**
+     * @notice Address of wrapped native token market
+     */
+    IVToken public immutable vWNativeToken;
 
     /**
      * @notice Constructor for NativeTokenGateway
-     * @param wrappedNativeToken_ Address of wrapped ether token contract
+     * @param wrappedNativeToken Address of wrapped native token contract
+     * @param vWrappedNativeToken Address of wrapped native token market
      */
-    constructor(IWrappedNative wrappedNativeToken_) {
-        ensureNonzeroAddress(address(wrappedNativeToken_));
-        wrappedNativeToken = wrappedNativeToken_;
+    constructor(IWrappedNative wrappedNativeToken, IVToken vWrappedNativeToken) {
+        ensureNonzeroAddress(address(wrappedNativeToken));
+        ensureNonzeroAddress(address(vWrappedNativeToken));
+
+        wNativeToken = wrappedNativeToken;
+        vWNativeToken = vWrappedNativeToken;
```

Comment:
`wrappedNativeToken` should be the underlying token of the `vWrappedNativeToken` token. We don't need to receive it as parameter

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/361/commits/85271d20aff3e6b9d37ef07faee22ef099a5b8e6